### ISSUE TITLE
Multi-selection ListBox widgets

### ISSUE THREAD

**Issue by ktritz** [chianti-atomic/ChiantiPy#197]
The ListBox widgets are multi-selection, which doesn't make sense for many of the choices (e.g. abund choice, line ratio, etc.), . There should be a keyword passed when it makes sense to activate multi-selection.

**Comment by kdere** [comment]
Yes, you are right.  However, for line ratios, if two lines are blended, you probably want to have the opportunity for multiple choices.  For abundances,  only one choice is appropriate
